.github/workflows/publish-arc2.yaml

```diff
@@ -19,12 +19,12 @@ on:
         type: boolean
         default: false
       publish_actions_runner_controller_2_chart:
-        description: 'Publish new helm chart for actions-runner-controller-2'
```

Comment:
do you want to also rename the workflow file? `publish-arc2.yaml`

Reply:
Yep good point, let me fix it

Reply:
This should be fixed now